### ISSUE TITLE
Could not determine java version from '9.0.1'.

### ISSUE THREAD

**Issue by rolljee** [Netflix/photon#215]
__What i've done so far__

```shell
cd /usr/local/bin
git clone https://github.com/Netflix/photon
cd photon
./gradlew clean
./gradlew build
./gradlew getDependencies
```

```shell
➜  photon git:(master) ✗ java --version
java 9.0.1
Java(TM) SE Runtime Environment (build 9.0.1+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)
```

```shell
./gradlew clean --stacktrace --debug
12:29:48.969 [ERROR] [org.gradle.BuildExceptionReporter]
12:29:49.020 [ERROR] [org.gradle.BuildExceptionReporter] FAILURE: Build failed with an exception.
12:29:49.022 [ERROR] [org.gradle.BuildExceptionReporter]
12:29:49.023 [ERROR] [org.gradle.BuildExceptionReporter] * What went wrong:
12:29:49.024 [ERROR] [org.gradle.BuildExceptionReporter] Could not determine java version from '9.0.1'.
12:29:49.029 [ERROR] [org.gradle.BuildExceptionReporter]
12:29:49.030 [ERROR] [org.gradle.BuildExceptionReporter] * Exception is:
12:29:49.035 [ERROR] [org.gradle.BuildExceptionReporter] java.lang.IllegalArgumentException: Could not determine java version from '9.0.1'.
12:29:49.036 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.api.JavaVersion.toVersion(JavaVersion.java:63)
12:29:49.036 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.api.JavaVersion.current(JavaVersion.java:72)
12:29:49.037 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:32)
12:29:49.038 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
12:29:49.039 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:206)
12:29:49.040 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:169)
12:29:49.041 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
12:29:49.041 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
12:29:49.043 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.Main.doAction(Main.java:33)
12:29:49.045 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
12:29:49.046 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
12:29:49.047 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
12:29:49.048 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
12:29:49.048 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
12:29:49.049 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:54)
12:29:49.050 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:35)
12:29:49.051 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
12:29:49.052 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
12:29:49.052 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
12:29:49.053 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
12:29:49.054 [ERROR] [org.gradle.BuildExceptionReporter] 	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
12:29:49.054 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:30)
12:29:49.055 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:129)
12:29:49.055 [ERROR] [org.gradle.BuildExceptionReporter] 	at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
12:29:49.057 [ERROR] [org.gradle.BuildExceptionReporter]
```

__I'm using  mac OS High sierra version 10.13.2__

So i've tried to `brew install gradle`( version 4.4 )

But i ran with an other error add the end of the build... 

```shell
> Configure project :
Inferred project: Photon, version: 4.6.0-SNAPSHOT


FAILURE: Build failed with an exception.

* What went wrong:
org/gradle/api/internal/project/AbstractProject
> org.gradle.api.internal.project.AbstractProject

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

I'm a bit lost so far, can anyone help me ? :-)

**Comment by MarcAntoine-Arnaud** [comment]
Have you try to build with docker ?
Or you can use directly: https://hub.docker.com/r/arnaudmarcantoine/photon/

**Comment by rolljee** [comment]
Hey, it's been a very long time since this issues was open. (I've forgotten that this was still open)
We have found our solution, we reinstalled all the environment and choose an other repo to download Java. so it worked for us luckily.

It's a good idea to use photon inside docker to avoid those errors. but our solution that use photon CLI as an Api already run in docker. I don't really now if it's a good idea to multiply docker inside a docker right ?
